user/how-to-guides/how-to-use-usb-devices.md

```diff
@@ -155,10 +159,26 @@ This should output something like:
 ../../../devices/pci-0/pci0000:00/0000:00:1a.0/usb3
 ```
 
+If the output format does not match this example, or you are unsure if it contains the correct BDF address, you can try finding the address using method 2 using the Qube Manager instead.
+
```

Comment:
I might be missing something, but if Method 2 always works reliably, while Method 1 sometimes gives the wrong output, then why even have two methods? Why not just scrap Method 1 and only present Method 2? (Of course, in that case, we wouldn't call it "Method 2," we should just refer to it as *the* recommended method.)

Reply:
Well, I didn't want to take out the original method because I don't know why it was recommended in the first place instead of just looking in the Qubes Manager GUI. Maybe there are cases in which its not possible to use the GUI or maybe at the time this was written it was not yet possible to see the device addresses in the Qubes Manager?

Reply:
I'm not sure either. Perhaps @marmarek can steer us in the right direction.

Reply:
The described method is about finding to which USB controller given device is connected, not about just identifying some USB controller. The Method 2 described below does not help with this at all ("you can probably guess").

As for not working reliably, it does work, it just needs a little clarification regarding situation in https://github.com/QubesOS/qubes-issues/issues/7711: if there are multiple addresses in the path, the _last_ one is the BDF of the controller. Taking example from that issue, given `../../../devices/pci0000:00/0000:00:1c.0/0000:01:00.0/usb1`, the answer is `01:00.0`, not `00:1c.0` (this one is a PCI bridge to which the controller is connected).